configs/components/dmidecode.rb

```diff
@@ -1,17 +1,22 @@
 component 'dmidecode' do |pkg, settings, platform|
-  pkg.version '2.12'
-  pkg.md5sum '02ee243e1ecac7fe0d04428aec85f63a'
-  pkg.url "http://download.savannah.gnu.org/releases/dmidecode/dmidecode-#{pkg.get_version}.tar.gz"
-  pkg.mirror "http://buildsources.delivery.puppetlabs.net/dmidecode-#{pkg.get_version}.tar.gz"
```

Comment:
Seems like we're losing the pkg.mirror setting in this merge-up. Was there a merge conflict that you had to manually resolve? If so, that's a great opportunity to add the pkg.mirror line back in.